documentation/modules/configuring-mtv-operator.adoc

```diff
@@ -66,6 +69,23 @@ spec:
 |`controller_block_overhead`
 |Fixed amount of additional space allocated in persistent block volumes. This setting is applicable for any `storageclass` that is block-based. It can be used when data, such as encryption headers, is written to the persistent volumes in addition to the content of the virtual disk. Note, this setting can only be changed using the CLI.
 |`0`
+
+
+|`vsphere_osmap_configmap_name`
+|Configuration map for vSphere source providers. This configuration map maps the incoming operating system of the incoming VM to a {virt} preference name. This configuration map needs to be in the namespace where `forklift` is deployed, that is, where `forklift-controller` is running.
+
+To see the list of preferences in your {virt} environment, open the {ocp-name} web console and click *Virtualization* -> *Preferences*.
+
+An authorized user can overwrite, add, or delete values from the default settings. Note, this setting can only be changed using the CLI.
```

Comment:
there are two ways to read this - (1) that the user can modify the specified config map; (2) that the user can modify this setting such that it will point to a different config map
both cases are possible, but which one did you meant here?

Reply:
I meant the first -- is that correct?

Reply:
it's correct, both are - but the second is obvious (it wouldn't make sense to have a config value for this if it wasn't possible to update it :) ). ok, I'd suggest to mention the CLI-only part above and be more explicit about referring to the first option here with something like:
```suggestion
An authorized user can overwrite, add, or delete values from the specified configuration map.
```

Reply:
@ahadas Fixed and also applied to the description of `ovirt_osmap_configmap_name` that follows.